modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidget.php

```diff
@@ -709,6 +709,16 @@ protected function buildContentTab(FieldItemListInterface $items, $delta, array
         $content_types_options = array_intersect_key($content_types_options,
           array_flip($allowed_content_types));
       }
+      $temp = [];
+      $content_types_options = array_reverse($content_types_options, TRUE);
+      if (array_key_exists('landing_page', $content_types_options)) {
+        $temp = ['landing_page' => $content_types_options['landing_page']];
+        unset($content_types_options['landing_page']);
+      }
+
+      if (!empty($temp)) {
+        $content_types_options = $temp + $content_types_options;
+      }
```

Comment:
@krakerag , part of the acceptance criteria is to have the checkboxes in this order: 

Order of checkboxes:

> Landing page
> 
> Publication (new)
> 
> News
> 
> Event (new)

That's why I have to reorder them.

Reply:
> Seems a complicated piece of code to assign them when we know the order, we should just set the order and move on, rather than doing a temp assignment to an array and add it to another array etc.

@krakerag , you mean, leave them by default? Alphabetically.

Reply:
No I mean literally just set them in the right order so the code is clear

`['landing_page', 'publication', 'news', 'event']`